### PR TITLE
Game Inis: Additional games requiring 'Immediate Mode' off

### PR DESCRIPTION
**Pull request by iwubcode**
I seem to have missed a few in my original Hybrid XFB PR.  I've added 3 more games to have Immediate Mode off:

- Xenoblade / Twilight Princess which have frame pacing issues
- Rhythm Heaven Fever which seems to have issues in some levels with it on

### CODE REVIEW

File: Data/Sys/GameSettings/SX4.ini [hunk 1]
```diff
@@ -18,3 +18,6 @@ EmulationIssues = The game randomly freezes.
 # Add action replay cheats here.
 
 [Video_Hacks]
+
+[Video_Hacks]
```
Comment (leoetlino) [inline Data/Sys/GameSettings/SX4.ini]: This line should not be required, as there is already a `[Video_Hacks]` just above

Comment (iwubcode) [reply Data/Sys/GameSettings/SX4.ini]: Whoops, copy-paste bug.  I'll fix it, thanks!

**Comment by JMC47** [comment]
@stenzek ^ that's the case where the hack we removed would be useful... even with locking, wouldn't it be better to have it working without XFB2RAM?

**Comment by iwubcode** [comment]
@JMC47 - looks like that's another case of partial-xfb?

**Comment by JMC47** [comment]
Yeah, but, the other two only happened on loading screens, which would be fine for locking to handle.  This one appears to be the whole game.

**Comment by JMC47** [comment]
I verified that vanilla Rogue Squadron 2 has bugs with XFB2RAM in-game (I didn't check our fix far enough apparently!) so, maybe we should disable it there as well?  When you get on IRC I'll try to debug it, but, it looks the same as the demo screenshots I was shown.

**Comment by iwubcode** [comment]
@JMC47 - likely won't be on tonight, not feeling well.  But next time I'm on, let's discuss RS2.  Also, we can talk about if it makes sense for some games to enable XFB2Ram by default for some games (Waldo and possibly Viewtiful Joe, I know a user reported it in the issue tracker).

File: Data/Sys/GameSettings/GZ2.ini [hunk 2]
```diff
@@ -17,3 +17,5 @@ EmulationIssues =
 [ActionReplay]
 # Add action replay cheats here.
 
+[Video_Hacks]
+ImmediateXFBEnable = False
```
Comment (LaosmAI) [inline Data/Sys/GameSettings/GZ2.ini]: image

**Comment by leoetlino** [comment]
What's the status of this PR?

**Comment by iwubcode** [comment]
@leoetlino - as far as I know, this is ready to go

**Pull request merged**